package.json

```diff
@@ -26,15 +26,9 @@
   ],
   "scripts": {
     "clean": "rm -rf lib && rm -rf dist && rm -f tsconfig.tsbuildinfo",
-    "build": "yarn build-lib && yarn build-types",
-    "build:release": "yarn clean && yarn build && yarn build-web",
-    "build-lib": "babel src -x .ts -d lib",
-    "build-types": "tsc --declaration --incremental --outDir lib --project tsconfig.build.json --emitDeclarationOnly",
-    "build-web": "webpack --mode production --entry ./lib/web.js --output ./dist/bls.min.js",
```

Comment:
please restore this

Reply:
We are not using the `build-web` script anywhere at the moment

Reply:
we use in build:release

Reply:
But build:release is not used anywhere

Reply:
ugh, it was used by cayman to deploy, I guess we need to modify realease script to use it